change/@ni-nimble-components-56124785-d76b-4bd0-b330-b457eb4ff9a0.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "major",
+  "comment": "Adding eight icons, renaming two, and replacing two. Clients using \"nimble-icon-calendar\" or \"nimble-icon-arrow-u-rotate-left\" must instead use \"nimble-icon-calendar-days\" and \"nimble-icon-arrow-u-up\" (respectively).",
```

I don't think folks looking at the git history / PR titles care as much about the number of icons vs what the icons are at a high level

```suggestion
  "comment": "Updates to calendar and arrow icons. Clients using \"nimble-icon-calendar\" or \"nimble-icon-arrow-u-rotate-left\" must instead use \"nimble-icon-calendar-days\" and \"nimble-icon-arrow-u-up\" (respectively).",
```